src/metabase/api/table.clj

```diff
@@ -66,14 +71,18 @@
                      :description             description))
     (api/check-500 (db/update! Table id, :visibility_type visibility_type))
     (let [updated-table (Table id)
+          driver (->> id table/table-id->database-id driver/database-id->driver) ;; is this the easy way to do this?
```

Comment:
You can just do `(driver/database-id->driver (:db_id updated-table))`. Otherwise you're doing an extra DB call here